jekyll/_cci2/orb-author.md

```diff
@@ -66,7 +66,7 @@ When designing your own orbs, make sure your orbs meet the following requirement
 * Match commands to executors - If you are providing commands, try to provide one or more executors in which they will run.
 * Use concise naming for your orb - Remember that use of your commands and jobs is always contextual to your orb, so you can use general names like "run-tests" in most cases.
 * Required vs. optional parameters - Provide sound default values of parameters whenever possible.
-* Avoid job-only orbs - Job-only orbs are inflexible. While these orbs are sometimes appropriate, it can be frustrating for users to not be able to use the commands in their own jobs. Pre and post steps when invoking jobs are a workaround for users.
+* Avoid job-only orbs - Job-only orbs are inflexible. While these orbs are sometimes appropriate, it can be frustrating for users to be not able to use the commands in their own jobs. Pre and post steps when invoking jobs are a workaround for users.
```

Comment:
```suggestion
* Avoid job-only orbs - Job-only orbs are inflexible. While these orbs are sometimes appropriate, it can be frustrating for users to be unable to use the commands in their own jobs. Pre and post steps when invoking jobs are a workaround for users.
```

Reply:
@rosieyohannan LGTM!!